src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -402,10 +402,27 @@ describe('Test suite for mappings on existing concepts', () => {
       },
     };
     const url = '1234';
+    const { value } = event.target;
     const instance = wrapper.find('EditConcept').instance();
     instance.updateEventListener({ target: { tabIndex: 0, name: INTERNAL_MAPPING_DEFAULT_SOURCE, value: '' } });
     instance.updateEventListener(event, url);
-    expect(instance.state.mappings[0].to_concept_code).not.toEqual(null);
+    expect(instance.state.mappings[0].to_concept_code).toEqual(value);
+  });
+
+  it('should call updateEventListener function on source change', () => {
```

Comment:
How is this related to the ticket functionality?

Reply:
It is meant to test for the event where a different source of a mapping is changed, the value of the "to_concept_code" is cleared because "to_concept_code" is equal to " ".

Reply:
Can you do your tests in terms of end user functionality?

Reply:
I have adjusted the test to stimulate user selection.

Reply:
Why should updateEventListener be called on source change?

Reply:
This is due to the fact that whenever a change is made e.g when the source of the mapping changes, updateEventListener is called to update the state.